cmake/FindMOAB.cmake

```diff
@@ -17,26 +17,30 @@ if (MOAB_LIBRARIES AND MOAB_INCLUDE_DIRS)
   # in cache already
   set(MOAB_FOUND TRUE)
 else (MOAB_LIBRARIES AND MOAB_INCLUDE_DIRS)
-  find_path(MOAB_INCLUDE_DIR NAMES MBCore.hpp
-    PATHS $ENV{HOME}/.local/include 
-        PATH_SUFFIXES include Include
+  find_path(MOAB_INCLUDE_DIR NAMES MBiMesh.hpp
```

Comment:
Great call on changing what file to look for. `MBCore.hpp` was removed in moab 4.8.2 which was about 2 years ago.

Reply:
Thanks!